vm/devices/storage/disk_crypt/block_crypto/src/lib.rs

```diff
@@ -0,0 +1,328 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Cryptography primitives for disk encryption.
+
+#![warn(missing_docs)]
+
+#[cfg(windows)]
+use bcrypt as sys;
```

Comment:
yay, another place in the codebase where we roll our own micro cross-platform encryption abstraction whoooo /s

not saying it needs to get solved in this PR, but its a shame we still don't have a single `support/native_crypto` or whatever

Reply:
I agree. If the existing ones were not junk and/or were in their own crates, I would have started with one of them.